src/Microsoft.OData.Core/ODataBatchUtils.cs

```diff
@@ -179,8 +179,8 @@ internal static void ValidateReferenceUri(Uri uri, IEnumerable<string> dependsOn
                 string baseUriString = UriUtils.UriToString(baseUri);
                 if (!uri.AbsoluteUri.StartsWith(baseUriString, StringComparison.Ordinal))
                 {
-                    throw new ODataException(Strings.ODataBatchReader_AbsoluteURINotMatchingBaseUri(
-                        uri.AbsoluteUri, baseUriString));
+                    // Skip validation if uri doesn't start with the baseUri.

```

Comment:
If the request specifies invalid ref to other operations, the request will result in error. Otherwise, it will be alright since validation would pass anyway.

---
In reply to: [197583353](https://github.com/OData/odata.net/pull/1195#discussion_r197583353) [](ancestors = 197583353)

Reply:
Okay thanks